image_width: 1600
image_height: 1200
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [856.77255,   0.     , 712.16903,
           0.     , 854.26373, 643.61137,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.316905, 0.093905, 0.001399, 0.001725, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [630.46033,   0.     , 722.76699,   0.     ,
           0.     , 624.9743 , 649.44361,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
